scripts/ci-build-azure-ccm.sh

```diff
@@ -43,8 +43,6 @@ setup() {
     export IMAGE_REGISTRY=${REGISTRY}
     pushd "${AZURE_CLOUD_PROVIDER_ROOT}" && IMAGE_TAG=$(git rev-parse --short=7 HEAD) && export IMAGE_TAG && popd
     echo "Image Tag is ${IMAGE_TAG}"
-    export AZURE_CLOUD_CONTROLLER_MANAGER_IMG=${IMAGE_REGISTRY}/${CCM_IMAGE_NAME}:${IMAGE_TAG}
```

Comment:
We no longer need to compose the `AZURE_CLOUD_CONTROLLER_MANAGER_IMG` and `AZURE_CLOUD_NODE_MANAGER_IMG` env vars because the templates that were expecting those env vars no longer exist.